helm/prometheus-rules/templates/platform/atlas/recording-rules/grafana-cloud.rules.yml

```diff
@@ -371,6 +371,10 @@ spec:
       record: aggregation:prometheus:memory_percentage
     - expr: sum(label_replace(container_memory_working_set_bytes{container='prometheus', namespace=~'.*-prometheus'}, "cluster_id", "$2", "pod", "(prometheus-)(.+)(-.+)")) by (cluster_id, cluster_type, customer, installation, pipeline, provider, region)
       record: aggregation:prometheus:memory_usage
+  - name: mimir.grafana-cloud.recording
```

Comment:
Do we want to add promxy to the prometehus one above?

Reply:
It wouldn't work because of the label_replace thing, and it shows memory per prometheus.

Reply:
Can we not add a row for promxy then like aggregation:promxy:memory_usage ?

Reply:
In any case that's peanuts https://giantswarm.grafana.net/explore/metrics/trail?from=now-1h&to=now&var-ds=000000006&var-filters=container%7C%3D%7Cpromxy-app&var-filters=installation%7C%3D%7Cgazelle&metricSearch=container&metric=aggregation:container:memory_usage_bytes&actionView=breakdown&layout=grid&var-groupby=$__all